static/translations/messages_en.json

```diff
@@ -199,5 +199,17 @@
   "Reactions": "Reactions",
   "No reactions": "No reactions",
   "See who reacted": "See who reacted",
-  "Only organization admins are allowed to post to this stream.": "Only organization admins are allowed to post to this stream."
+  "Only organization admins are allowed to post to this stream.": "Only organization admins are allowed to post to this stream.",
+  "Set a status": "Set a status",
+  "Connecting...": "Connecting...",
+  "User status": "User status",
+  "Muted": "Muted",
+  "Log in with GitLab":"Log in with GitLab",
+  "No topics found":"No topics found",
+  "What's your status?":"What's your status?",
+  "📅 In a meeting": "📅 In a meeting",
+  "🚌 Commuting": "🚌 Commuting",
+  "🤒 Out sick": "🤒 Out sick",
+  "🌴 Vacationing":"🌴 Vacationing",
+  "🏠 Working remotely":"🏠 Working remotely"
```

Comment:
These should follow the same formatting as the other entries: a single space after the colon.